routing/dht/routing.go

```diff
@@ -84,7 +84,7 @@ func (dht *IpfsDHT) GetValue(ctx context.Context, key key.Key) ([]byte, error) {
 	ctx, cancel := context.WithTimeout(ctx, time.Minute)
 	defer cancel()
 
-	vals, err := dht.GetValues(ctx, key, 16)
+	vals, err := dht.GetValues(ctx, key, (KValue/2)+1)
```

Comment:
i performed a seance and the ghost of alan turing said that (K/2)+1 was a good value.